gluster/block/README.md

```diff
@@ -1,31 +1,37 @@
 # glusterblock Volume Provisioner for Kubernetes 1.5+
```

Comment:
Why is there a readme overhaul in a commit called "More divided code for guster-block and heketi mode"? This should be a commit of its own. (Commit messages should always reflect what a commit actually does... ;-).